validator/src/main/java/org/apache/skywalking/plugin/test/agent/tool/validator/entity/SegmentForRead.java

```diff
@@ -176,8 +176,8 @@ public void setComponentName(String componentName) {
             this.componentName = componentName;
         }
 
-        public void setError(String error) {
-            isError = error;
+        public void setIsError(String isError) {
+            this.isError = isError;
```

Comment:
Do you mean current checking not working? IsError seems a right name from Java style.

Reply:
Yesterday, I found the error status is not a valid option. After I run the unit test, the status is like the following:
![image](https://user-images.githubusercontent.com/31562192/92339985-3f684400-f0eb-11ea-9d40-cd403d77d0dd.png)